<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QuickQash POS</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>QuickQash POS - Touch friendly demo</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>64</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="salesArea">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QWidget" name="leftPanel">
         <property name="minimumSize">
          <size>
           <width>420</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="leftLayout">
          <item>
           <widget class="QPushButton" name="btn_products">
            <property name="text">
             <string>Products</string>
            </property>
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>64</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_pos">
            <property name="text">
             <string>POS</string>
            </property>
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>64</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_settings">
            <property name="text">
             <string>Settings</string>
            </property>
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>64</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_printqueue">
            <property name="text">
             <string>Print Queue</string>
            </property>
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>64</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="rightPanel">
         <layout class="QVBoxLayout" name="rightLayout"/>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
